client/components/lessons/LessonNavigation/LessonNavigation.tsx

```diff
@@ -0,0 +1,48 @@
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+import HorizontalSeparator from '~/components/HorizontalSeparator';
+import { getAdjacentLessons } from '~/services/DataModel';
+import styles from './LessonNavigation.module.scss';
+
+export default function LessonNavigation(
+  { lessonId }: PropsWithChildren<{ lessonId: string }>,
+) {
+  const [adjacentLessons, setAdjacentLessons] = useState([]);
+
+  useEffect(() => {
+    setAdjacentLessons(getAdjacentLessons(lessonId));
+  }, []);
+
+  if (adjacentLessons.length === 0) return null;
+
+  const [previousLesson, nextLesson] = adjacentLessons;
+
+  return (
+    <>
+      <HorizontalSeparator />
```

Comment:
Why not `border-top`? If not, maybe change color to `darker-white`? 

The default color doesn't look so good.

Reply:
I added border-top instead.